components/common/src/util/path.rs

```diff
@@ -116,7 +116,7 @@ pub async fn interpreter_paths() -> Result<Vec<PathBuf>> {
                             }
                         }
                         None => {
-                            install::type_erased_start(&mut ui::UI::default_with_env(),
+                            install::type_erased_start(&mut ui::NullUi::new(),
```

Comment:
Do we need to have a `UI` here? I switched to `NullUi` to allow spawning the `install` future (`UI` is not `Send`). If we want to keep the `UI` I could probably have two versions of the function.